pallets/capacity/src/lib.rs

```diff
@@ -203,6 +203,14 @@ pub mod pallet {
 		/// A staker may not retarget more than MaxRetargetsPerRewardEra
 		#[pallet::constant]
 		type MaxRetargetsPerRewardEra: Get<u32>;
+
+		/// The fixed size of the reward pool in each Reward Era.
+		#[pallet::constant]
+		type RewardPoolEachEra: Get<BalanceOf<Self>>;
+
+		/// the percentage cap per era of an individual Provider Boost reward
+		#[pallet::constant]
+		type RewardPercentCap: Get<Permill>;
```

Comment:
I wonder if these should eventually be government settable values instead of constants, but either way it is better this way for now.